src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources.mdx

```diff
@@ -92,6 +90,29 @@ for monitoring host entities using the host metrics receiver for the OpenTelemet
 * [`host.image.id`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/host)
 * [`host.image.version`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/host)
 
+### Redis instances [#redis]
+
+A redis instance is synthesized using data emitted from the collector [redis receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/redisreceiver). Unfortunately, there are currently no semantic conventions for redis instances, or identifying attributes in the metrics the redis receiver emits. Therefore, users are required to manually include the attributes described below.
+
+Refer to the redis example in [collector for infrastructure monitoring](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-intro/) for more details.
+
+#### Required attributes [#redis-required-attributes]
+
+* [`server.address`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/server/): The redis instance name you see in the UI comes from this value.
```

Comment:
See entity definition for reference: https://github.com/newrelic/entity-definitions/blame/c13e7a5d7150556c3980a64cd44047f70f3bdca0/entity-types/infra-redisinstance/definition.yml#L7-L42